designer/src/components/field-editor.tsx

```diff
@@ -88,6 +94,29 @@ export const FieldEditor = ({
   const [selectedFormId, setSelectedFormId] = useState<string | null>(null);
 
   const fieldComponent = field['component-name'];
+  const protection = field['component-parameters'].protection || 'none';
+  const isHidden = field['component-parameters'].hidden || false;
+  const isRequired = field['component-parameters']?.required || false;
+
+  const notebookMetadata = useAppSelector(state => state.notebook.metadata);
+
+  const isDerivedFromSet = Boolean(notebookMetadata['derived-from']);
```

Comment:
Would be good to display this field somewhere in the INFO panel as a read-only value.  'This template is derived from XXXX, some restrictions may apply when editing fields.'